assets/scss/modules/_tabs.scss

```diff
@@ -3,6 +3,80 @@
   overflow: hidden;
   position: relative;
 
+  .tabs-nav__tab {
+    margin-bottom: 0;
+    display: block;
+    padding: 0.5em;
+    white-space: nowrap;
+    outline: none;
+    z-index: 0;
+    box-sizing: border-box;
+    position: relative;
+    border-top: 1px solid $border-colour;
+    transition: background .25s ease-in-out;
+
+    &:focus,
+    &:hover,
+    &:active {
+      background-color: $govuk-grey-3;
+    }
+
+    &:visited {
+      color: $govuk-blue-colour;
+    }
+
+    .count {
+      background-color: $red;
+      color: $white;
+      padding: 0 em(5);
+    }
+
+    @include media(tablet) {
+      height: em(53);
+      display: inline-block;
+      margin: 0 em(3) 0 0;
+      padding: 0.7em 0.8em;
+      border-top: none;
+      border-bottom: 6px solid white;
+    }
+
+  }
+
+  li.tabs-nav__tab {
+    margin-bottom: 0;
+  }
+
+  a.tabs-nav__tab {
+
+    &:focus,
+    &:hover,
+    &:active {
+      background-color: $govuk-grey-3;
+    }
+
+  }
+
+  .tabs-nav__tab--active {
+    z-index: 2;
+    color: black; // there is a variable for $black but it's in govuk_elements, which will be removed
```

Comment:
probably use `$black` for the moment and this can be a simple clean up task?
`$white` below is using the gov stuff still right?